resources/assets/js/projects/title.vue

```diff
@@ -61,11 +61,10 @@ export default{
             setTimeout(() => location.href = this.redirectUrl, 2000);
         },
         deleteProject() {
-            let confirmed = confirm(`Do you really want to delete the project ${this.project.name}?`);
-
-            if (confirmed) {
+            let inputproject = prompt(`Are you scure you want to delete "${this.project.name}"? Please type the project name to confirm.`);
```

Comment:
```suggestion
            let inputproject = prompt(`Are you sure you want to delete "${this.project.name}"? Please type the project name to confirm.`);
```
Please fix the typo.